rules/docker-platform-monitor.mk

```diff
@@ -6,7 +6,7 @@ DOCKER_PLATFORM_MONITOR_DBG = $(DOCKER_PLATFORM_MONITOR_STEM)-$(DBG_IMAGE_MARK).
 
 $(DOCKER_PLATFORM_MONITOR)_PATH = $(DOCKERS_PATH)/$(DOCKER_PLATFORM_MONITOR_STEM)
 
-$(DOCKER_PLATFORM_MONITOR)_DEPENDS += $(LIBSENSORS) $(LM_SENSORS) $(FANCONTROL) $(SENSORD) $(LIBSWSSCOMMON) $(PYTHON3_SWSSCOMMON)
+$(DOCKER_PLATFORM_MONITOR)_DEPENDS += $(LIBSENSORS) $(LM_SENSORS) $(FANCONTROL) $(SENSORD) $(LIBSWSSCOMMON) $(PYTHON3_SWSSCOMMON) $(IPMITOOL)
```

Comment:
why this pr added IPMITOOL? not related to bookworm upgrade?

Reply:
Since, the ipmitool available in the bookworm debian repo had an issue, the ipmitool was built locally with the necessary fix in place and installed in the host as part of #17878.
Hence, the locally built ipmitool with the fix is installed in pmon instead of installing using apt.